Components/Snackbar/snackbar.css

```diff
@@ -0,0 +1,139 @@
+/* ///////baseline///// */
+
+.baseline {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  font-size: large;
+  background-color: var(--baseline-background);
+  width: fit-content;
+  border-radius: 5px;
+  padding: 5px 10px;
+}
+
+.baseline-para {
+  color: var(--background-color);
+}
+
+.baseline-retry {
+  display: flex;
+  color: var(--snackbar-retry);
+  align-items: center;
+  padding: 10px;
+}
+
+.baseline-cross {
+  color: var(--background-color);
+  display: flex;
+  align-items: center;
+  padding: 10px 15px;
+  font-size: 2rem;
+  font-weight: 900;
+  border-radius: 50%;
+  margin-top: auto;
+  margin-bottom: auto;
+}
+
+.baseline-cross:hover {
+  cursor: pointer;
+  background-color: var(--baseline-hover);
+  border-radius: 50%;
+}
+
+.baseline-retry:hover {
+  cursor: pointer;
+}
+
+/*//////////// LEADING  /////////////*/
+
+.leading {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: large;
+  background-color: var(--baseline-background);
+  width: fit-content;
+  border-radius: 5px;
+  margin-top: 3rem;
+  padding: 5px 10px;
+}
+
+.leading-para {
+  color: var(--background-color);
+}
+
+.leading-retry {
+  display: flex;
+  color: var(--snackbar-retry);
+  align-items: center;
+  padding: 10px;
+}
+
+.leading-cross {
+  color: var(--background-color);
+  display: flex;
+  align-items: center;
+  padding: 10px 15px;
+  font-size: 2rem;
+  font-weight: 900;
+  border-radius: 50%;
+  margin-top: auto;
+  margin-bottom: auto;
+}
+
+.leading-cross:hover {
+  cursor: pointer;
+  background-color: var(--baseline-hover);
+  border-radius: 50%;
+}
+
+.leading-retry:hover {
+  cursor: pointer;
+}
+
+/*/////////// Stacked ////////////*/
+
+.stacked {
+  display: flex;
+  font-size: large;
+  justify-content: center;
+  flex-direction: column;
+  background-color: var(--baseline-background);
+  width: fit-content;
+  border-radius: 5px;
+  margin-top: 3rem;
+  padding: 10px;
+}
+
+.stacked-para {
+  color: var(--background-color);
+  padding-top: 10px;
+}
+
+.stacked-label {
+  display: flex;
+  margin-left: auto;
+  color: var(--snackbar-retry);
+  align-items: center;
+  padding: 10px;
+}
+
+.stacked-cross {
+  color: var(--background-color);
+  display: flex;
+  align-items: center;
+  padding: 10px 15px;
+  font-size: 2rem;
+  font-weight: 900;
+  border-radius: 50%;
+}
+
+.stacked-cross:hover {
+  cursor: pointer;
+  background-color: var(--baseline-hover);
+  border-radius: 50%;
+}
+
+.stacked-label:hover {
+  cursor: pointer;
+}
```

Comment:
You can use all the snackbar hover classes at one place and style them all together 
eg:
.stacked-label : hover,
.leading-retry:hover{
cursor : pointer}

Same can be done for other also